internal/pkg/cli/svc_init_test.go

```
@@ -115,10 +116,13 @@ func TestSvcInitOpts_Validate(t *testing.T) {
 				m.mockStore.EXPECT().GetApplication("phonetool").Return(&config.Application{}, nil)
 			},
 			mockFileSystem: func(mockFS afero.Fs) {
-				mockFS.MkdirAll("hello", 0755)
+				err := mockFS.MkdirAll("hello", 0755)
+				if err != nil {
+					return 
+				}
```

prob fine to ignore this error